packages/rpc-spec/src/rpc.ts

```diff
@@ -63,7 +63,7 @@ function makeProxy<TRpcMethods, TRpcTransport extends RpcTransport>(
 
 function createPendingRpcRequest<TRpcMethods, TRpcTransport extends RpcTransport, TResponse>(
     rpcConfig: RpcConfig<TRpcMethods, TRpcTransport>,
-    pendingRequest: RpcRequest<TResponse>,
+    pendingRequest: PendingRpcApiRequest<TResponse>,
```

Comment:
Nice. I think the relationship makes sense. Give me a pending _RPC API_ request, and I'll turn it into a pending _RPC_ request using the transport.